docs/docs/namespaces.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Namespaces
+route: /namespaces
+---
+
+# Namespaces
+
+When using Tanka, namespaces are handled slightly different compared to
+`kubectl`, because environments offer more granular control than contexts used
+by `kubectl` do.
```

Comment:
```suggestion
by `kubectl`.
```

Reply:
If I remember it right, 'prepositions' are something never to end a sentence with.